ios/brave-ios/Sources/Brave/Frontend/Rewards/BraveRewards.swift

```diff
@@ -213,32 +213,32 @@ public class BraveRewards: NSObject {
     }
   }
 
-  /// Report that a page has loaded in the current browser tab, and the HTML is available for analysis
-  ///
-  /// - note: Send nil for `adsInnerText` if the load happened due to tabs restoring
-  ///         after app launch
+  /// Report that a page has loaded in the current browser tab, and the
+  /// text/HTML content is available for analysis.
   func reportLoadedPage(
     redirectChain: [URL],
     tabId: Int,
     html: String,
-    adsInnerText: String?
+    innerText: String?
   ) {
     guard let url = redirectChain.last else {
       return
     }
 
     tabRetrieved(tabId, url: url, html: html)
-    if let innerText = adsInnerText, ads.isServiceRunning() {
+    if ads.isServiceRunning() {
       ads.notifyTabHtmlContentDidChange(
```

Comment:
@aseren can you please confirm `notifyTabHtmlContentDidChange` will be called if a user has not joined Brave Rewards so that search result ad conversions work as expected? Thanks

Reply:
I confirm that HTML content change is notified. There is a log:
```
[ads] Tab id ... HTML content changed
```